### ISSUE TITLE
Mobile gestures on fullscreen

### ISSUE THREAD

**Issue by philwastell** [artpolikarpov/fotorama#313]
Feature request...

Would it possible to add in native style image zooming (i.e. pinch to zoom) when in fullscreen mode? So rather than enlarging the entire screen, just the image would zoom leaving the controls in the correct position.

**Comment by scooterlord** [comment]
Yep, similar to #138. Art? Are you still working on this one? :)

**Comment by artpolikarpov** [comment]
Another essential gesture — swipe to close.

**Comment by yura3d** [comment]
@artpolikarpov Hi! Are there any news? Since 2014 (when the issue was created) desktop/mobile devices ratio in the web changed dramatically, on my projects I see that almost 65% of visits are made from mobiles. Users complain that they cannot use pinch or double tap zoom as they used to native Android/iOS apps.

**Comment by a22432** [comment]
It would be great to see this improvement in fotorama
